app/components/EarthquakeChart/EarthquakeChart.tsx

```diff
@@ -0,0 +1,117 @@
+import { EarthquakeEvent } from '@/utils/fetchEarthquakes';
+import React, { useEffect, useRef, useState } from 'react';
+import { Bar } from 'react-chartjs-2';
```

Comment:
I'm going to move away from this lib in the next chunk of work, and hopefully get this page squared away so I can comfortable deploy *something*